comp/otelcol/ddflareextension/impl/server.go

```diff
@@ -112,7 +113,11 @@ func newServer(endpoint string, handler http.Handler) (*server, error) {
 	r := mux.NewRouter()
 	r.Handle("/", handler)
 
-	r.Use(validateToken)
+	// no easy way currently to pass required bearer auth token to OSS collector;
+	// skip the validation if running inside a separate collector
+	if !auth {
+		r.Use(validateToken)
+	}
```

Comment:
cc @truthbk are we ok with this ?

Reply:
added a card for backlog https://datadoghq.atlassian.net/browse/OTEL-2226